### DISCUSSION TITLE
Juntar duas contas em uma - Merge two accounts into one

### DISCUSSION THREAD

**Discussion by fabiohcnobre** [community/community#30090] [Other Feature Feedback, Questions, & Ideas] [answered]
Bom dia,
Tenho duas contas uma com o email (pessoal)
E uma com email  (empresa) eu consigo juntar as duas contas em uma conta somente?

Quero ficar somente com a pessoal, porem queria passar meus comites e meus acessos da conta empresa.
Att
Fabio

____ EN
Good Morning,
I have two accounts one with the email (personal)
And one with email (company) can I join the two accounts into one account?

I want to stay only with the staff, but I wanted to pass on my committees and my company account access.
Att
fabio

**Comment by jfullstackdev** [comment, marked as answer]
simply add the email that you want to use and you're done, GitHub encourages one account with multiple emails linked for different commits

**Reply by fabiohcnobre** [reply]
But I already have both accounts registered. I need to join.

**Reply by jfullstackdev** [reply]
carefully follow the steps

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/merging-multiple-personal-accounts

**Reply by fabiohcnobre** [reply]
How to do Merging multiple personal accounts?

**Reply by jfullstackdev** [reply]
the idea is, you will be deleting the other accounts after you moved all of your existing project to your target account, then add all those emails to the target account too, so that the commits will be restored

**Comment by jricciardi** [comment]
Hi @fabiohcnobre! Just an FYI, GitHub Discussions has a machine translation feature in Beta. You should be able to post in Portuguese without having to worry about supplying your own translation 😄 

https://user-images.githubusercontent.com/31521880/185702879-e1dc7d5d-53aa-408c-b1fc-d076d91cfcc5.mov

**Reply by fabiohcnobre** [reply]
Nossa que maravilha isso eu gostei ... Obrigado pela dica!
